src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -134,6 +135,16 @@ class SidebarScreen extends Component {
                                         onSelected: () => Navigation.navigate(ROUTES.IOU_BILL),
                                     },
                                 ] : []),
+                                ...(Permissions.canUseFreePlan(this.props.betas) ? [
+                                    {
+                                        icon: NewWorkspace,
+                                        iconWidth: 44,
```

Comment:
Is the width arbitrary? The icon overflows a bit, Not visible in our white bg though. It would be better, If we can make it such that it doesn't cut.

<img width="1121" alt="Screenshot 2021-06-17 at 11 22 24 AM" src="https://user-images.githubusercontent.com/32012005/122339197-639c5f80-cf5e-11eb-99b8-d536a21c3fe4.png">

Reply:
True, it's 46px instead of 44px!

Reply:
Thanks!